### PR TITLE
NAS-107495 / 20.10 / Nas 107495

### PR DESCRIPTION
**Pull request by lola27**

### CODE REVIEW

**Comment by william-gr** [comment]
Could you please post a few screenshots with how it looks like now?
Its harder for us who are not familiar with the code to visualize what it will be ;)

**Comment by lola27** [comment]
Screen Shot 2020-09-30 at 8 52 07 AM

**Comment by lola27** [comment]
Screen Shot 2020-09-30 at 8 52 23 AM

**Comment by lola27** [comment]
Screen Shot 2020-09-30 at 8 52 39 AM

**Comment by william-gr** [comment]
Thanks Lola. Looks good to me. Lets try to keep this only for SCALE for the time being. I will update tickets.

The target select box is not aligned with the others.
Are we going to make it a slide panel?

**Comment by lola27** [comment]
@william-gr The target select box will be aligned if the tooltip is not empty. (we may need to retouch the CSS later).
Maybe we will move the wizard into a sliding panel, but I am not sure. we may need to discuss it.

**Review by william-gr — APPROVED** [review]

File: src/app/helptext/sharing/iscsi/iscsi.ts [hunk 1]
```diff
@@ -347,6 +347,9 @@ export const helptext_sharing_iscsi = {
   usefor_placeholder: T("Sharing Platform"),
   usefor_tooltip: T("Choose the platform that will use this share. The associated options are applied to this share."),
 
+  target_placeholder: T('Target'),
+  target_tooltip: T(''),
```
Comment (Mrt134) [inline src/app/helptext/sharing/iscsi/iscsi.ts:351]:
```suggestion
  target_tooltip: T('Create a new Target or choose an existing target for this share.'),
```

**Review by Mrt134 — APPROVED** [review]
Added a basic help text suggestion.

**Review by dkmullen — APPROVED** [review]
lgtm

**Pull request merged**